app/src/main/java/org/gnucash/android/db/adapter/AccountsDbAdapter.java

```diff
@@ -1176,7 +1176,7 @@ public boolean isFavoriteAccount(String accountUID){
 
                 Transaction transaction = new Transaction(GnuCashApplication.getAppContext().getString(R.string.account_name_opening_balances));
                 transaction.setNote(getAccountName(accountUID));
-                transaction.setCurrencyCode(currencyCode);
```

Comment:
I'm not sure I follow you here. The `transactions.currency_code` column is still being set with the currency code from the Commodity in Transaction. So, for now, it shouldn't be a problem.

We should probably get rid of the column with the db migration, but I think it's used in some view. So it might not be as easy as it seems.